src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/Context/FormattingContext.InitialContextFinder.cs

```diff
@@ -79,12 +80,7 @@ private List<IndentBlockOperation> GetInitialIndentBlockOperations(SyntaxToken s
                             {
                                 _formattingRules.AddIndentBlockOperations(list, n);
                                 foreach (var element in list)
-                                {
-                                    if (element != null)
-                                    {
-                                        operations.Add(element);
-                                    }
-                                }
+                                    operations.Add(element);
```

Comment:
❔ Can we not just use `operations.AddRange(list)` here?

Reply:
we could, but it woudl also allocate due to the IENumerable.GetEnumerator call.  Should i add an overload that takes a SegmentedList.

Reply:
This was already impacting other cases in the pull request, so I submitted #51331. We can make this change here in parallel and it's safe to assume that #51331 will merge before this one. I'm already offsetting the performance numbers to account for that change.

Reply:
done.